receiver/filelogreceiver/README.md

```diff
@@ -44,10 +44,9 @@ It can also be easily configured to tail and parse any structured or unstructure
 Receiver Configuration
 ```yaml
 receivers:
-  stanza:
-    operators:
-      - type: file_input
-        include: [ /var/log/myservice/*.json ]
+  filelog:
+    include: [ /var/log/myservice/*.json ]
+    parsers:    
```

Comment:
>Renames operators to parsers. I believe this term conveys the intention of this sub-pipeline better than operators, even though we may end up supporting some operations that are not truly just "parsing". For example, we may want to be able add/delete/move fields.

If we do this will these non-parser operators be placed under "parsers" or we will have a different subkey to put them under?

Reply:
You make a good point. Splitting operators into multiple buckets would make the order or operations ambiguous. It also would be difficult to code/maintain.

I think that `parsers` is a good term in the vast majority of cases, but perhaps it's best to leave this alone for now.